NBitcoin/RPC/WalletCreateFundedPSBTResponse.cs

```diff
@@ -0,0 +1,12 @@
+using NBitcoin.BIP174;
+
+namespace NBitcoin.RPC
+{
+	public class WalletCreateFundedPSBTResponse
+	{
+		public PSBT PSBT { get; internal set; }
+		public Money Fee { get; internal set; }
+		// -1 if no change output.
+		public int ChangePos { get; internal set; }
```

Comment:
Can you instead use `int?` and affect `null` if the RPC call send back `-1`? It is more easy to understand as C# dev.